scientific_library/tvb/simulator/plot/plotter.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
```

Comment:
This type of main entrance to plotting might deserve to stay inside tvb_multiscale only

Reply:
After discussing with @dionperd we agreed to keep this here.
Then adding the license at the top will be needed.